tests/Model/BSONArrayTest.php

```diff
@@ -4,9 +4,9 @@
 
 use MongoDB\Model\BSONArray;
 use MongoDB\Model\BSONDocument;
-use MongoDB\Tests\TestCase;
+use MongoDB\Tests\FunctionalTestCase;
```

Comment:
I changed this from `TestCase` to `FunctionalTestCase` to access `assertSameDocument()`. Alternatively, I could add `assertSameDocument()` and `normalizeBSON()` to  `TestCase`.